source/extensions/filters/http/lua/wrappers.cc

```diff
@@ -52,6 +52,27 @@ int HeaderMapWrapper::luaGet(lua_State* state) {
   }
 }
 
+int HeaderMapWrapper::luaGetAtIndex(lua_State* state) {
+  const char* key = luaL_checkstring(state, 2);
+  const auto index = luaL_checknumber(state, 3);
+  const auto header_value = headers_.get(Http::LowerCaseString(key));
```

Comment:
I know we still have "auto" all over the place in this file. But, let's use a more specific type for these?